nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutUDP.java

```diff
@@ -58,7 +58,7 @@ public class TestPutUDP {
     private final static char CONTENT_CHAR = 'x';
     private final static int DATA_WAIT_PERIOD = 50;
     private final static int DEFAULT_TEST_TIMEOUT_PERIOD = 10000;
-    private final static int LONG_TEST_TIMEOUT_PERIOD = 100000;
+    private final static int LONG_TEST_TIMEOUT_PERIOD = 30000;
```

Comment:
The failure mode of the load test was to send 500 messages, of which ~2-3 were dropped.  The 497 successful connections were received in a couple of seconds, then the test waits for the remaining failed packets, which will never be received.

Adjusting this timeout down to a number that matches expected behavior.  If it doesn't succeed in 30 seconds, it likely won't succeed in 100 seconds.